_posts/2018-06-13-hackillinois-2019-mission.markdown

```diff
@@ -1,43 +1,38 @@
 
-We _create an enviroment_  in that our event is not defined by a simple "code for a prize" model. We plan to pack the entire event with content for attendees, from UIUC and over a dozen other universities, so that they can have a unique experience throughout the weekend. HackIllinois carries development, social, and networking responsibilities, and these establish the three pillars —  _education, collaboration, and mentorship_ — of our event. The theme this year follows the trail that has been set forth by our past directors and staff, but we wanted to assert our post _within the Open Source community_. We fully believe that our event betters the Open Source Community: by bringing together new students, continuing development on projects we believe in, and showcasing the talent pool that we have to offer.
+# Something Borrowed
 
-More specifically, thie year we want to make HackIllinois even more **exciting**. We are putting as much emphasis as we can on making this event more than 36 hours of coding; simply learning about open source is not enough in 2019. It will be the immersive experience of teamwork, guidance, and excitement that hackers have come to expect out of our staff. We will run interactive workshops, present lively mini-events, and host top industry talent for our weekend in February. The Open Source experience will be unique from the past: exciting to both the OSS gurus and those who have not yet explored this region of the technology world.
+We are always open to feedback and new ideas. If you have participated in our event or any hackathon, conference, get-together, Urbana Sweetcorn Festival, or ANYTHING where you found a fun or engaging project, activity, or meal, do not hesitate to suggest it to us. We are still in the brainstorming phases for the event and love to hear from you all, so reach out to us! Working together and getting advice from informed individuals is our style. #(OUR HASHTAG HERE)
 
+There are some pilot ideas we will be exploring over the following months as planning continues, so let us know on contact@hackillinois.org if you have suggestions, feedback, or just want to share your experiences with us. We're always happy to hear from you! We will continue to make more announcements on our blog, right here, and on our social media.
 
-## Something Borrowed:
+Likewise, we will be attending conferences and hackathons alike to get a good sense of what is currently being developed in the world around us. If you see somebody in a HackIllinois staff shirt, please engage with us! We genuinely enjoy having conversations with our attendees, and it may be a much easier forum to discuss and get ideas across than over an email. Additionally, if you are (or will soon become) an Illinois student who'd like to get involved in this process and join our team, we will make a call for staff applications in the fall, so stay tuned :).
```

Comment:
after "join our team," maybe just say "look out for the HackIllinois meetings in the fall to learn more about how to get involved" or something like that 
if there are incoming students coming, don't want to scare them away with the idea of applications // seems like higher barrier to entry

Reply:
excellent point, changed it to that